pkg/util/fsync/fsync.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package fsync
```

Comment:
Perhaps something like `atomicfs` would be a better name? `fsync` is the name of the function used to achieve atomicity, rather than the purpose of this package.